netkan/Dockerfile

```diff
@@ -1,6 +1,7 @@
 FROM python:3.7 as base
 RUN useradd -ms /bin/bash netkan
 ADD . /netkan
+RUN chown -R netkan:netkan /netkan
 WORKDIR /netkan
```

Comment:
This isn't necessary, in production this path ceases to exist and in dev we'll either use the volume mount under home (via docker-compose) or the workspace (from within vs-code).

Reply:
Now obsolete thanks to `TemporaryDirectory()`